### PR TITLE
PRM Automated Test-LPS-172479|Master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LPS-172479](https://issues.liferay.com/browse/LPS-172479)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 1]
```diff
@@ -118,6 +118,53 @@ definition {
...
+				dateFormat = "MMddYYYY",
+				dateRange = "+185",
+				fieldName = "endDate");
+		}
+		
+		task ("Then a feedback message will appear") {
+		AssertElementPresent(
+			key_feedbackMessage = "End date must be less than six month after start date",
+			locator1 = "Form#FEEDBACK_MESSAGE");
+		}
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:167]: Hi @MGabriellaS, here you can use the same pattern that @mateussantes23 used on his PR:
```
task ("Then an error message will be displayed") {
			PRMMDFRequest.viewErrorMessage(errorMessage = "Start date need to be after today");
}
```
Just the error message is different.

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase (hunk 1 above)
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase]: ok I will do it

**Comment by AfonsoHSantos** [comment]
giphy

**Comment by AfonsoHSantos** [comment]
Hi @MGabriellaS, thanks for working on it! The code looks good to me.  I'm forwarding it!

**Comment by AfonsoHSantos** [comment]
ci:forward

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**